server/controllers/article.controller.js

```diff
@@ -204,5 +204,25 @@ class ArticleController {
     await sendMail(mailOption);
     return Response(res, 200, 'Article shared successfully');
   }
+
+  /**
+   * Delete article
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} response
+   */
+  static async deleteArticle(req, res) {
+    const {
+      params: { slug }, user: { id }
+    } = req;
+
+    const article = await Article.destroy({
```

Comment:
When deleting an article does the comments, threaded comments, and tags get deleted too? or just the article?

Reply:
Will implement that right away